torchao/quantization/__init__.py

```diff
@@ -37,6 +39,9 @@
     "compute_error",
     "Int4WeightOnlyGPTQQuantizer",
     "Int4WeightOnlyQuantizer",
+    "WeightOnlyInt4Linear",
+    "Int8DynActInt4WeightLinear",
+    "Int8DynActInt4WeightQuantizer",
```

Comment:
Maybe we shouldn't expose these if they weren't exposed before, since these are meant to be legacy APIs?